docs/tutorials/SpectralFit.ipynb

```diff
@@ -13683,99 +3382,251 @@
-      "text/plain": [
-       "<Figure size 640x480 with 1 Axes>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
     }
    ],
    "source": [
-    "# Propagate the errors to the \"evaluate_at\" method of the spectrum\n",
-    "results = like2.results\n",
+    "results = like.results\n",
     "\n",
     "print(results.display())\n",
     "\n",
     "parameters = {par.name:results.get_variates(par.path)\n",
-    "               for par in results.optimized_model[\"source\"].parameters.values()\n",
-    "               if par.free}\n",
+    "              for par in results.optimized_model[\"source\"].parameters.values()\n",
+    "              if par.free}\n",
     "\n",
     "if 'l' in parameters:\n",
     "    del parameters['l']\n",
     "if 'b' in parameters:\n",
     "    del parameters['b']\n",
+    "if 'ra' in parameters:\n",
```

Comment:
Can you remind me why you need this extra linea of code? I think we talked about it already, but can't remember. It looks hard to maintain.

Reply:
In general, if the uncertainty on the location of the source is comparable or greater than the angular resolution, setting the coordinates free and including them during the error propagation is the correct thing to do. It seems to me that this is rather a bug. 

For this PR, I rather leave the code using all free parameters during the propagation by default, since that the correct thing to do. Instead, can you please open an issue elaborating on the error that you get? That'll have to be solved later.

That said, a somewhat better way to obtain only the parameters of the spectrum is:

```
parameters = {par.name:results.get_variates(par.path)
              for par in results.optimized_model.source.spectrum.main.shape.parameters.values()
              if par.free}
```

This picks up the parameters specifically to the spectrum, independently of how the sky location is parametrized.